var/spack/repos/builtin/packages/coreneuron/package.py

```diff
@@ -37,7 +37,8 @@ class Coreneuron(CMakePackage):
     url      = "https://github.com/BlueBrain/CoreNeuron"
 
     version('develop', git=url, submodules=True)
-    version('0.14.1', git=url, submodules=True, preferred=True)
+    version('0.15', git=url, tag='0.15', submodules=True)
+    version('0.14', git=url, tag='0.14', submodules=True)
```

Comment:
@iomaganaris : I think we changed version `0.14.1` with tag `0.14` for some reason...could you check the log of this recipe  change?

Reply:
I think we bumped the version there to rebuild the modules with disabled `nrn_timeout`. However, there was not created a new tag in `CoreNeuron`.